src/esbmc/options.cpp

```diff
@@ -59,6 +59,12 @@ const struct group_opt_templ all_cmd_options[] = {
       boost::program_options::value<std::string>()->value_name("cname"),
       "set contract name"},
    }},
+#endif
+#ifdef ENABLE_PYTHON_FRONTEND
+  {"Python frontend",
+   {
+     {"show-python-ast", NULL, "show Python AST"},
```

Comment:
I wonder whether we can re-use the existing --parse-tree-{only,too} options instead of adding a new but semantically equivalent option.

Reply:
I think so. I'll take a look.

Reply:
Done. Thank you.